assets/js/components/Table/CollapsibleTableRow.jsx

```diff
@@ -27,7 +27,7 @@ function CollapsibleTableRow({
       </tr>
       {collapsibleDetailRenderer && (
         <tr
-          className={`overflow-y-hidden ${
+          className={`bg-gray-100 overflow-y-hidden ${
```

Comment:
@jagabomb A comment on this. 
Changing this will change our collapsible table rows, as this would be a default color now.
I'm not sure we want this...
You can check how it looks in `DatabasesOverview` in storybook for example.
Just for your information, as I'm a bit afraid of doing this "generic" changes for specific things